back/app/controllers/web_api/v1/files_v2_controller.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+class WebApi::V1::FilesV2Controller < ApplicationController
```

Comment:
I'm not sure about the use of `FilesV2` as a name (particularly when you see it in the context of `web_api/v1`, but I cannot think of anything more appropriate